lib/babe/epoch_handler_test.go

```diff
@@ -8,14 +8,17 @@ import (
 	"testing"
 	"time"
 
+	"github.com/ChainSafe/gossamer/dot/types"
 	"github.com/ChainSafe/gossamer/lib/crypto/sr25519"
 	"github.com/ChainSafe/gossamer/pkg/scale"
 
 	"github.com/stretchr/testify/require"
 )
 
 func TestNewEpochHandler(t *testing.T) {
-	testHandleSlotFunc := func(epoch, slotNum uint64, authorityIndex uint32, proof *VrfOutputAndProof) error {
+	testHandleSlotFunc := func(epoch, slotNum uint64, authorityIndex uint32,
+		preRuntimeDigest *types.PreRuntimeDigest,
+	) error {
```

Comment:
nit dis looks better imo but feel free to discard

```suggestion
		preRuntimeDigest *types.PreRuntimeDigest) error {
```